Dockerfile

```diff
@@ -1,22 +1,27 @@
 FROM node:10-alpine
-LABEL Description="This image builds and runs the Human-Connection Web Application Frontend" Vendor="Human-Connection gGmbH" Version="0.0.1" Maintainer="Human-Connection gGmbH (developer@human-connection.org)"
+LABEL Description="Web Frontend of the Social Network Human-Connection.org" Vendor="Human-Connection gGmbH" Version="0.0.1" Maintainer="Human-Connection gGmbH (developer@human-connection.org)"
 
-# expose the app port
+# Expose the app port
 EXPOSE 3000
 
-ARG WORKDIR=/HC-WebApp
```

Comment:
I would not add `nitro` here. That name will be removed once we drop the legacy web app.